docs/install_linux.md

```diff
@@ -13,30 +13,19 @@ If none of our official binaries, packages, repositories, nor community sources
 
 ### Debian, Ubuntu Linux (apt)
 
-Install (Using apt-key):
+:warning: This will only work for the deb-based distributions we [explicitly support](/.github/workflows/releases.yml#L83) and the [architectures we officially support](/.goreleaser.yml#L27).
```

Comment:
Why couldn't we allow anyone with a Debian derivative to fetch our deb package, regardless of what their `lsb_release -cs` reports? I.e. instead of evaluating `lsb_release -cs`, we put some static string in sources.list, like "stable".

Reply:
we can do that, though it can lead to annoying behavior with certain distributions especially around distro upgrade time. I'll put `stable` for now but leave a note about how a more specific codename can be used if desired.